transformation-scripts/semopenalex-authors.py

```diff
@@ -304,13 +317,27 @@ def transform_gz_file(gz_file_path):
                     if not author_created_date is None:
                         author_graph.add((author_uri, DCTERMS.created, Literal(author_created_date, datatype=XSD.date)))
 
+                    # affiliations (rdf-star)
+                    affiliations = json_data['affiliations']
```

Comment:
please access json object using the following approach; 
```suggestion
                    affiliations = json_data.get('affiliations')
```
<img width="1263" alt="Screenshot 2024-06-07 at 12 17 36 PM" src="https://github.com/metaphacts/semopenalex/assets/15888835/bacebcf1-ce26-40f1-aea5-035cf921eb26">